xchange-stream-service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -529,4 +541,7 @@ public void setSocksProxyHost(String socksProxyHost) {
   public void setSocksProxyPort(Integer socksProxyPort) {
     this.socksProxyPort = socksProxyPort;
   }
+  public void setDisableNettyReconnect(boolean noRetry) {
```

Comment:
I'm not a fan of this naming of things as "setDisabled". It's a bit ambiguous as to what you're actually setting. The name also exposes the fact that it's Netty unnecessarily, where as this is an implementation detail specific to the fact that this is a NettyStreamingService. Imagine that it implements StreamingService, and the api comes from there... it wouldn't say Netty in the Interface. 

Can we instead make it, `setAutoReconnect(boolean autoReconnect)`, to make it perfectly clear that a true value will enable auto reconnect, where as a false value will make it not auto reconnect?

Reply:
We can use AutoReconnect instead